specs/trust-store-trust-policy.md

```diff
@@ -185,7 +255,7 @@ Trust policy for the scenario where ACME Rockets uses some artifacts signed by W
 - `strict` : Signature verification is performed at `strict` level, which enforces all validations. If any of these validations fail, the signature verification fails. This is the recommended level in environments where a signature verification failure does not have high impact to other concerns (like application availability). It is recommended that build and development environments where images are initially created, or for high assurance at deploy time use `strict` level.
 - `permissive` : The `permissive` level enforces most validations, but will only logs failures for revocation and expiry. The `permissive` level is recommended to be used if signature verification is done at deploy time or runtime, and the user only needs integrity and authenticity guarantees.
 - `audit` : The `audit` level only enforces signature integrity if a signature is present. Failure of all other validations are only logged.
-- `skip` : The `skip` level does not fetch signatures for artifacts and does not perform any signature verification. This is useful when an application uses multiple artifacts, and has a mix of signed and unsigned artifacts. Note that `skip` cannot be used with a global scope (`*`), the value of `registryScopes` MUST contain fully qualified registry URL(s).
+- `skip` : The `skip` level does not perform any signature verification. This is useful when an application uses multiple artifacts, and has a mix of signed and unsigned artifacts. Note that `skip` cannot be used with a global scope (`*`).
```

Comment:
Suggestion - Note that `skip` cannot be used with a global scope, i.e `registryScopes` set to `*` in a Trust Policy for OCI artifacts, or if `globalPolicy` is set to `true` in a Trust Policy for Blobs.